R/plot-construction.r

```diff
@@ -38,6 +47,17 @@
 #' # This can be useful to return from a function.
 #' base + list(subset(mpg, fl == "p"), geom_smooth())
 "+.gg" <- function(e1, e2) {
+  UseMethod("add_gg")
```

Comment:
This should be `add_gg(e1, e2)`, I think

Reply:
```suggestion
  add_gg(e1, e2)
```